Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -7,7 +7,11 @@ import Foundation
 
 @objc(ChannelDTO)
 class ChannelDTO: NSManagedObject {
-    @NSManaged var cid: String
+    @NSManaged private var cid: String
+    var channelId: ChannelId {
+        try! .init(cid: cid)
```

Comment:
I would recommend to use try? with default variable like 

`ChannelId(cid: cid) ?? .unknown`  or `.invalid`